src/apps/integrations/service/integrations.py

```diff
@@ -125,4 +125,23 @@ async def retrieve_integration(self, applet_id, integration_type) -> Integration
                     configuration=future_integration,
                 )
             case _:
-                raise UnsupportedIntegrationError(type=integration_type)
+                raise UnsupportedIntegrationError(integration_type=integration_type)
+
+    async def delete_integration(self, applet_id, integration_type) -> Integration:
```

Comment:
Calling delete on a previously deleted integration results in an unhandled exception
![CleanShot 2024-09-21 at 09 13 50@2x](https://github.com/user-attachments/assets/e653f02b-4caa-441d-be05-a8ade6bd1f39)